<launch>
	<include file="$(find turtlebot_bringup)/launch/minimal.launch"/>
	<include file="$(find turtlebot_navigation)/launch/amcl_demo.launch">
		<arg name="map_file" value="$(find tourbot)/files/map/floor_map.yaml"/>
	</include>
	<include file="$(find turtlebot_rviz_launchers)/launch/view_navigation.launch"/> 

	<node pkg="sound_play" type="soundplay_node.py" name="soundplay_node"/>
	<node pkg="usb_cam" type="usb_cam_node" name="usb_cam">
		<param name="pixel_format" value="yuyv"/>
	</node>
 </launch>
